extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -652,12 +665,20 @@ export class AmpA4A extends AMP.BaseElement {
           });
         }))
         // some() returns an array of which we only need a single value.
-        .then(returnedArray => returnedArray[0]);
+        .then(returnedArray => returnedArray[0], () => {
+          // Rejection occurs if all keys for this provider fail to validate.
+          return Promise.reject(
+              `All keys ${keyInfoSet.serviceName} failed to verify`);
```

Comment:
Nit: Feels like the message needs another word.  "All keys for...", "All keys from..."...?